src/Ability.js

```diff
@@ -1,5 +1,8 @@
 /** @module */
 
+/**
+ * Represents a players used ability
+ */
```

Comment:
combine please

Reply:
done all except for utils in case we add more functions in there later on